config/apiserver/kustomization.yaml

```
@@ -0,0 +1,8 @@
+resources:
+- apiserver.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: apiserver
+  newName: quay.io/operator-framework/catalogd-server
+  newTag: latest
```

Based on discussion in the other PR?
```suggestion
  newTag: devel
```

---

I was going to wait and make the necessary changes based on whichever PR gets in first